types/express/index.d.ts

```diff
@@ -99,7 +100,7 @@ declare namespace e {
     interface Request<P extends core.Params = core.ParamsDictionary> extends core.Request<P> { }
     interface RequestHandler<P extends core.Params = core.ParamsDictionary> extends core.RequestHandler<P> { }
     interface RequestParamHandler extends core.RequestParamHandler { }
-    export interface Response extends core.Response { }
+    export interface Response<ResBody = any> extends core.Response<ResBody> { }
```

Comment:
The core.Response already has generics. 
`export interface Response<ResBody = any> extends http.ServerResponse, Express.Response `

This will uplevel the generics to be use by Express Response.